<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Michael L. Bernauer</title>
        <meta name="description" content="A framework for easily creating beautiful presentations using HTML">
        <meta name="author" content="Michael L. Bernauer">
        <meta name="apple-mobile-web-app-capable" content="yes">
        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
        <link rel="stylesheet" href="../reveal/css/reveal.css">
        <link rel="stylesheet" href="../reveal/css/theme/serif.css" id="theme">
        <!-- Code syntax highlighting -->
        <link rel="stylesheet" href="../reveal/lib/css/zenburn.css">
        <!-- Printing and PDF exports -->
        <script>
            var link = document.createElement( 'link' );
            link.rel = 'stylesheet';
            link.type = 'text/css';
            link.href = window.location.search.match( /print-pdf/gi ) ? '../reveal/css/print/pdf.css' : '../reveal/css/print/paper.css';
            document.getElementsByTagName( 'head' )[0].appendChild( link );
        </script>
        <!--[if lt IE 9]>
        <script src="lib/js/html5shiv.js"></script>
        <![endif]-->
    </head>
    <body>

        <div class="reveal">

            <!-- Any section element inside of this container is displayed as a slide -->
            <div class="slides">
<section data-markdown>
<script type="text/template">
# Biomedical Informatics
<center>
  <cite>
    <i>
      Michael Bernauer<br>
      PharmD Candidate 2016<br>
      University of New Mexico College of Pharmacy
    </i>
  </cite>
</center>
</script>
</section>
<section>
<section data-markdown>
<script type="text/template">
# What is Biomedical Informatics (BMI)
</script>
</section>
<section data-markdown>
<script type="text/template">
## Definition

### American Medical Informatics Association (AMIA)
BMI is the interdisciplinary field that studies and pursues the effective 
use of biomedical data, information, and knowledge for scientific inquiry, problem
solving, and decision making, motivated by efforts to improve human health.
<cite>(*Kulikowski C., et al. JAMIA. 2012*)</cite>

Field that is concerned with the optimal use of information, often aided by the use
of technology and people, to improve individual health, health care, public health, and biomedical
research.
</script>
</section>
<section data-markdown>
<script type="text/template">
## Interdisciplinary
* Draws from fields such as biology, research and clnical medicine, computer science, data science and analytics, statistics, engineering
* Fuzzy boundaries/conceptual overlap between competencies/disciplines

<center><img src='./bmi-domains.svg' width='650px'/></center>
<small>Reproduced from American Medical Informatics Association (http://www.amia.org/about-amia/science-informatics)</small>
</script>
</section>
<section data-markdown>
<script type="text/template">
## The Many Domains of BMI
* **Clincial Informatics**: informatics applied to healthcare (nusring, pharmacy, dentistry, pathology) 
* **Bioinformatics**: informatics in cellular/molecular biology, $1,000 genome era (genomics, chemiformatics)
* **Imaging Informatics**: informatics as applied to imaging, storage and retrieval (radiology)
  * Computer vision/Image analysis and segmentation
* **Consumer Health Informatics**:
  * Remote Monitoring:
    * Wireless Compliance Monitoring
    * Activity trackers/wireless accelerometry
  * Direct-To-Consumer Gentic Testing
    * 23andMe, deCODEme, FamilyTreeDNA
* **Research Informatics**: use of informatics to facilitate biomedical and health research (clinical/translational research)
* **Public Health Informatics**: includes surveillance, reporting and health promotion
  * Surveillance:
    * FDA Adverse Event Reporting System (FAERS)/Vaccine Adverse Event Reporting System (VAERS)
    * PulseNET
* **Health IT Management**: management of medical records
</script>
</section>
</section>
<section>
<section data-markdown>
<script type="text/template">
# Consumer Health Informatics

## (Remote Monitoring)
</script>
</section>
<section data-markdown>
<script type="text/template">
## Activity trackers/Wireless accelerometry
Increasingly being used by clinicians as tool for assessing activity and mobility in outpatient setting
  * Monitoring mobility after major surgery (*Cook D., et al. The Annals of Thoracic Surgery. 2013*)
  * Monitoring physical activity in cardiac patients (*Alharbi M., et al. Eur J Prev Cardio. 2016*)
  * Measurement of gait abnormalities for the purpose of identifying patients with idiopathic normal pressure hydrocephalus (iNPH) (*Galgani S., et al. Cureus. 2015*)
  * 83 hits for 'fitbit in [clincialtrials.gov](http://clinicaltrials.gov)

<center><img src='fitbit.svg' width='400px' /></center>
<small>http://clinicaltrials.gov</small>
</script>
</section>
<section data-markdown>
<script type="text/template">
## SmackCAP (Pressure Analysis Company)
* UNM startup
* Skullcap worn during contact sports
* Wireless impact sensors detect and track impacts to the head
* ImPACt scorecard for longitudinal assessment

<center><img src='https://epactechnologies.com/img/front-page.jpg' width='500px'/></center>
<small>http://epactechnologies.com</small>
</script>
</section>
<section data-markdown>
<script type="text/template">
## Wireless Medication Compliance Monitoring
### Problem
* **Adherence**: act of filling/refilling prescriptions on time
* **Compliance**: act of taking medications on schedule or taking medications as prescribed
* Estimated that 50-55% of children and their families do not follow treatment plans as prescribed (*Bovet P. et al. Bulliten of the World Health Organization. 2002*)
* Non-compliance may result in MDRO and increase relapse, morbidity, mortality and rehospitalization resulting in increased healthcare costs

### Solution
* Build a device that allows compliace monitoring and enforcement
* Spark Core $\mu$Controller
  * ARM 32-bit Cortex CPU
  * 128KB flash memory
  * 12 bit ADC
  * IEEE 802.11 b/g compliant wifi module (CC3000)
  * 500mA output current
  * 18 IO
</script>
</section>
<section data-markdown>
<script type="text/template">
## Wireless Medication Monitor
<center><img src="wireless-monitor.svg" width='800px'/></center>
</script>
</section>

<section data-markdown>
<script type="text/template">
# Consumer Health Informatics
## (Direct-To-Consumer Genetic Testing)
</script>
</section>
<section data-markdown>
<script type="text/template">
## DTC SNP Genotyping

### 23andMe, deCODEme, FamilyTreeDNA
* Allow consumers to purchase SNP genotypes
* Products include "carrier status" and "wellness" reports
* Disease risk reports pending future FDA approval

### OpenSNP
* Open source repository consisting of DTC SNP data
* Additionally collects phenotypic information
* Fitbit API for integration of activity tracker data
</script>
</section>
<section data-markdown>
<script type="text/template">
## OpenSNP

<center><img src='open-snp-stats.png' width='700px'/></center>
<small>https://opensnp.org/statistics</small>
</script>
</section>
</section>
<section>
<section data-markdown>
<script type="text/template">
## Interests
* **Data mining**: extraction of patterns and knowledge from large amounds of data. Considered to be a critical part of knowledge discovery in databases.
  * FAERS, PubMed
* **Machine Learning**: subfield of artificial intelligence (AI) concerned with algorithms that allow computers to learn.
  * Can be broken in to supervised and unsupervised
* **Natural Language Processing (NLP)**:
  * Text summarization
  * Information extraction
  * Word embedding: word2vec
    * Skip-gram
    * Continuous Bag of Words (CBOW)
  * Information retreival
</script>
</section>
<section data-markdown>
<script type="text/template">
## Machine Learning: Unsupervised
Goal is to reveal natural structure or patterns within unlabeled, often high-dimensional data
* **Cluster analysis**: Hierarchical, k-Means
* **Principle Component Analysis (PCA)**
* **Singular Value Decomposition (SVD)**
* **Latent Semantic Analysis**
* **Latent Dirichlet Allocation**
* **Word Embedding**: skip-gram, CBOW
</script>
</section>
<section data-markdown>
<script type="text/template">
## Machine Leraning: Supervised
Learned properties of the data are derived through training on labled training set. Predictions can then be made on unseen data.
* **Regression methods**: prediction of continuous variable
  * k-Nearest Neighbors
  * Linear Regression
  * Logistic Regression
  * Artificial Neural Networks
* Classification methods**: prediction of class lables/categories
  * k-Nearest Neighbors
  * Linear Discriminant Analysis
  * Logistic Regression
  * Support Vector Machines (SVM)
  * Classification and Regression Trees
  * Naive Bayes Classifier
  * Artificial Neural Networks
    * Shallow/Deep architecture
</script>
</section>
</section>
<section>
<section data-markdown>
<script type="text/template">
# Projects
</script>
</section>
<section data-markdown>
<script type="text/template">
## Identification of MRSA PFGE Strain Type using Machine Learning
### Background
* PFGE is an electrohporetic molecular typing method used in epidemiologic surveillance of MRSA
* Electrophoresis of restricted DNA results in characteristic fingerprint
* Current methods of analysis (Gel Compar II) use hierarchical clustering based on pairwise similarity to classify organisms
* Current methods require preprocessing and manual intervention (lane detection, preprocessing)

### Objective
* Study the feasability of using machine learning algorithms as alternatives to hierarchical clustering in determining
PFGE strain type of methicillin-resistant Staphylococcus *aureus* isolates
</script>
</section>
<section data-markdown>
<script type="text/template">
## Identification of MRSA PFGE Strain Type using Machine Learning
### Methods
* 70 PFGE images were obtained containing ~ 20 fingerprints per image
* Lanes were extracted using k-means clustering and normalized
* Training set was created by applying PFGE class labels
* Final training set contained 843 fingerprints
* PCA was used as feature selection/dimensionality reduction of the fingerprint array
* SVM was implemented using 10-fold cross validation
* Grid search was used to select hyperparameters for kNN, RF and aNN
</script>
</section>
<section data-markdown>
<script type="text/template">
## Identification of MRSA PFGE Strain Type Machine Learning
### Results
* aNN and SVM performed best with overall classification accuracy of 90.0% and 88.5% respectively
* kNN and RF performed similarly with accuracies of 86.1% and 86.9% respectively

### Limitations
* Big $p$, little $n$: may result in overfitting/high variance
* Unbalance dataset (preponderace of USA300)

### Conclusion
Supervised learning algorithms such as SVM and aNN provide feasable alternatives to traditional hierarchical clustering methods. It is expected that accuracy
will improve with more training data.
</script>
</section>
</section>
<section>
<section data-markdown>
<script type="text/template">
## FDA Adverse Event Reporting System (FAERS)
### Background
* FAERS is a public databse containing information on adverse event and medication error reports submitted to FDA
* Designed to support post-marketing saftey surveillance of drug and therapeutic biologic products
* Reporting is voluntary by healthcare professionals and required by manufacturers
* Evaluation of potential safety concern may result in updates to labeling information, restricted use of med, or product removal from market

### Objective
* Develop as set of open source tools mining adverse drug events using data from the FAERS
</script>
</section>
<section data-markdown>
<script type="text/template">
## FDA Adverse Event Reporting System (FAERS)
### Methods
* Develop tools for normalizing drugnames ([drugstandards](https://github.com/mlbernauer/drugstandards))
  * drugstandards works by creating a set a mappings from brand name to generic using [DrugBank](http://www.drugbank.ca/) (Wishart DS., et al. Nucleic Acids Res. 2006) and [ChEMBL](https://www.ebi.ac.uk/chembl/) (Bento AP., et al. Nucleic Acids Res. 2014)
  * Jaro-Winkler similarity is used to handle misspellings and ranges from 0 to 1
* Standard data mining algorithms were implemented (Proportional Reporting Ratio, Reporting Odds Ratio, Mult-item Gamma Poisson Shrinker)

$$PRR = \frac{P(ae|drug)}{P(ae|-drug)}$$
</script>
</section>
<section data-markdown>
<script type="text/template">
## Adverse Event Matrix
<center><img src="faers-mat.png" width='600px'/></center>

</script>
</section>
<section data-markdown>
<script type="text/template">
## FDA Adverse Event Reporting System (FAERS)
### Future Directions
* Drug-drug interactions using association rules
* Cluster analysis
* Matrix factorization/"topic modeling"
</script>
</section>
</section>

        </div>
        <script src="../reveal/lib/js/head.min.js"></script>
        <script src="../reveal/js/reveal.js"></script>
        <script>
            // Full list of configuration options available at:
            // https://github.com/hakimel/reveal.js#configuration
            Reveal.initialize({
                controls: true,
                progress: true,
                history: true,
                center: true,
                transition: 'convex', // none/fade/slide/convex/concave/zoom
                // Optional reveal.js plugins
                dependencies: [
                    { src: '../reveal/lib/js/classList.js', condition: function() { return !document.body.classList; } },
                    { src: '../reveal/plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: '../reveal/plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: '../reveal/plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
                    { src: '../reveal/plugin/zoom-js/zoom.js', async: true },
                    { src: '../reveal/plugin/notes/notes.js', async: true },
                    { src: '../reveal/plugin/math/math.js', async: true}
                ]
            });
            Reveal.configure({backgroundTransition: 'zoom'});
        </script>
    </body>
</html>
