<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Michael L. Bernauer</title>
        <meta name="description" content="A framework for easily creating beautiful presentations using HTML">
        <meta name="author" content="Michael L. Bernauer">
        <meta name="apple-mobile-web-app-capable" content="yes">
        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
        <link rel="stylesheet" href="../reveal/css/reveal.css">
        <link rel="stylesheet" href="../reveal/css/theme/black.css" id="theme">
        <!-- Code syntax highlighting -->
        <link rel="stylesheet" href="../reveal/lib/css/zenburn.css">
        <!-- Printing and PDF exports -->
        <script>
            var link = document.createElement( 'link' );
            link.rel = 'stylesheet';
            link.type = 'text/css';
            link.href = window.location.search.match( /print-pdf/gi ) ? '../reveal/css/print/pdf.css' : '../reveal/css/print/paper.css';
            document.getElementsByTagName( 'head' )[0].appendChild( link );
        </script>
        <!--[if lt IE 9]>
        <script src="lib/js/html5shiv.js"></script>
        <![endif]-->
    </head>
    <body>

        <div class="reveal">

            <!-- Any section element inside of this container is displayed as a slide -->
            <div class="slides">
<section data-markdown>
<script type="text/template">
# Pharmacogenomics

[_Michael L. Bernauer_](http://mlbernauer.com/)

*University of New Mexico*
</script>
</section>
<section data-markdown>
<script type="text/template">
## Pharmacogenomics
* Employs tools for assessing genetic determinants of drug response
* Historically a phenotype-to-genotype approach was taken
* Now, with genomics, we can take a reverse; genotype-to-phenotype approach
in which polymorphisms serve as the starting point
</script>
</section>
<section data-markdown>
<script type="text/template">
## Importance of Pharmacogenomics
* Individuals response to a drug depends on complex interplay between environmental factors
and genetic factors
* For monogenic traits (G6DP, CYP2D6, TPMT), its often possible to predict
phenotype from genotype
* Prospective genotype determinations may result in the ability to prevent ADRs
* Defining multigenic contributors to drug response is much more challenging
* Multigenic phenotypes with large number of candidate genes require large sample size
to produce sufficient statistical power to solve the 'multigenic' problem
</script>
</section>
<section data-markdown>
<script type="text/template">
## Basics of Pharmacogenoics
* **Autosomal:** trait/allele that is located on autosome (i.e. not sex-linked)
* **Recessive:** phenotype only expressed if individual is homozygous for nonfunctional allele; will not present in heterozygotes
* **Co-dominant:** phenotype will present as intermediate if individual is heterozygous for nonfunctional allele
* **Dominant:** this trait will present phenotypically in both heterozygous or homozygous individuals
* Even within a single gene, a vast array of polymorphisms may exist (promoter, coding, noncoding, completely inactivating, modisty modifying)
* Each polymophism may differentially affect a measured trait
* A polymorphism with modest effect will only be observed in homozygous inidviduals
* A polymorphism with complete loss of function will have large effect and may be observed in heterozygotes
* Most traits are multigenic, not monogenic thus it is difficult to tell exactly how a given gene polymorphism will
influence the phenotypic trait
</script>
</section>
<section>
<section data-markdown>
<script type="text/template">
## Types of gene variants
* Polymorphism: variation in DNA sequence present in more than 1% of the population
* Two major types of sequence variation exist:
    * Single nucleotide polymorphisms (SNPs)
    * Insertion/deltions (indels)
</script>
</section>
<section data-markdown>
<script type="text/template">
## SNPs
* A single point mutation in which one nucleotide (ATCG) is substituted for another
* Occurs in >1% of population are SNPs
* May exist in coding regions (exons) or noncoding regions (introns, promoters)
* **Non-synonymous (missense):** result in amino acid substitutions
* **Synonymous (sense):** preserve amino acid sequence (wobble codons)
* **Nonsense:** result in early termination via introduction of stop codon
* Some synonymous polymorphism have been associated with alterations in phenotype; i.e. ABCB1 C3435T has
is synonymous but the new codon is translated at slower rate altering folding of final protein
</script>
</section>
<section data-markdown>
<script type="text/template">
## Indels
* Involves the insertion or deltion of bases
* Short repeats in promoter can alter expression levels
* May result in addition or subtraction of amino acids in protein
</script>
</section>
<section data-markdown>
<script type="text/template">
## Copy number variations (CNV)
* CNVs may cause gene duplication which increase that genes expression
* May cause deletions resulting in absence of gene expression
* Observed in TPMT, ABCB1, CYP450s, CBS, UGT, GST
</script>
</section>
</section>
<section data-markdown>
<script type="text/template">
## Haplotypes
* Defined as a series of allels found at a linked locus on a chromosome
* Individuals have two haplotypes for each gene; one maternal and one paternal
* Haplotypes represent the constellation of variations that occur together for the gene on each
chromosome
</script>
</section>
<section data-markdown>
<script type="text/template">
## Linkage Equilibrium/Disequilibrium
* Linkage equilibrium: genotype at one locus is independent of genotype at second locus
* Linkage disequilibrium: genotypes at two locus are not indipendent of one another
* Complete linkage disequilibrium: genotypes at two loci always occur together
</script>
</section>
<section data-markdown>
<script type="text/template">
## Haploview
![Haploview](http://openi.nlm.nih.gov/imgs/512/30/3338756/3338756_pone.0034741.g003.png)
</script>
</section>
<section data-markdown>
<script type="text/template">
## Ethnic Diversity
* Cosmopolitan polymorphisms are those polymorphisms present in all ethnic groups
* Population (race/ethnic) specific polymorphisms are consistent with geographical isoluatin of human populations and are often neutral (no advantage
or disadvantage)
</script>
</section>
<section data-markdown>
<script type="text/template">
## Monogenic vs. Multigenic
* Population frequency distribution of monogenic traits may be trimodal with relatively distinct separation among low, intermediate, and high
activity indidivudals
* Population frequency distribution for multigenic traits is unimodal-skewed with no distinct differences among groups
</script>
</section>
<section data-markdown>
<script type="text/template">
## Databases
</script>
</section>
<section data-markdown>
<script type="text/template">
| Name | URL | Description |
| --- | --- | --- |
| PharmGKB | [www.pharmgkb.org](www.pharmgkb.org) | Genotype and phenotype data related to drug response |
| EntrezSNP | [www.ncbi.nlm.nih.gov/SNP](www.ncbi.nlm.nig.gov) | SNPs and frequencies |
| HGVbase | [www.hgvbaseg2p.org](www.hgvbaseg2p.org) | Genotype/phenotype associations |
| HuGE Navigator | [www.hugenavigator.net](www.hugenavigator.net) | Literature annotations for genotype/phenotype associations |
| OMIM | [www.ncbi.nlm.nih.gov/sites/entrez/?db=OMIM](www.ncbi.nlm.nih.gov/sites/entrez/?db=OMIM) | Human genes and genetic disorders |
| HapMap | [www.hapmap.org](www.hapmap.org) | Genotypes, frequency, linkage data for variants |
| UCSC Genome Browswer | [http://genome.ucsc.edu](http://genome.ucsc.edu) | Sequence of the human genome; variant alleles |
| Novartis | [http://symatlas.gnf.org/SymAtlas/](http://symatlas.gnf.org/SymAtlas/) | Gene expression data for human genes |
| Broad Institute Software | [link](http://www.broad.mit.edu.libproxy.unm.edu/science/software/science/software/software) | Software tools for the analysis of genetic studies |
</script>
</section>
<section data-markdown>
<script type="text/template">
## GWAS SNP database
[http://www.ebi.ac.uk/gwas/docs/downloads](http://www.ebi.ac.uk/gwas/docs/downloads)
</script>
</section>
<section data-markdown>
<script type="text/template">
# Pharmacogenetic Phenotypes
Candidate genes for therapeutic and adverse response can be divided into three categories: _pharmacokinetic_, _receptor/target_, and _disease modifying_.
</script>
</section>
<section data-markdown>
<script type="text/template">
## Pharmacokinetic phenotypes
* 15-25% of all medicines in use have been shown to be substrates for CYP2D6
* Seven variant alleles account for well over 90% of the 'poor metabolizer' low-activity alleles for this gene
* Some individuals cary stable duplications of CYP2D6; 'ultra-rapid' metabolizers have up to 13 copies
* Deficient CYP2D6 phenotype results in antidepressant/antipsychotic toxicity (catabolized by enzyme), lack of analgesic effect of codeine, lack of active tamoxifen
</script>
</section>
<section data-markdown>
<script type="text/template">
# Genes of Drug Metabolism and Transport
</script>
</section>
<section data-markdown>
<script type="text/template">
## Drug Metabolism and Transport 
</script>
</section>
<section data-markdown>
<script type="text/template">
| Gene | Drugs | Response affected |
| --- | --- | --- |
| CYP2C9 | Tolbutamide, warfarin, phenytoin, NSAIDs | Anticoagulant effect of warfarin | 
| CYP2C19 | Omeprazole, propranolol, phenytoin, clopidogrel | Peptic ulcer response of omeprazole; Cardiovascular events after clopidogrel |
| CYP2D6 | Atomoxetine, dextromethorphan, fluoxetine, tamoxifen | Tardivedyskinesia from antipsychotics, narcotic SEs, codeine efficacy |
| CYP3A4/3A5/3A7 | Macrolides, cyclosporine, tacrolimus, CCBs, midazolam, lidocaine | Efficacy of immunosuppressive effects of tacrolimus |
</script>
</section>
<section data-markdown>
<script type="text/template">
## Drug Metabolism and Transport 
| Gene | Drugs | Response affected |
| --- | --- | --- |
| GSTM1, GSTT1, GSTP1 | Several anticancer agents | Decrease response in breast cancer, more toxicity |
| TPMT | Mercaptopurine, thioguanine, azathioprine | Thiopurine toxicity and efficacy |
| UGT1A1 | Irinotecal | Irinotecan toxicity |
| ABCB1 | HIV protease inhibitors, digoxin | Decreased CD4 response, decreased digoxin AUC |
| UGT2B7 | Morphine | Morphine plasma level |
| SLC01B1 | Statins, methotrexate, ACEIs | Statin plasma levels, mypoathy; methotrexate levels, mucositis |
</script>
</section>
<section data-markdown>
<script type="text/template">
## Drug metabolism and transport 
| Gene | Drugs | Response affected |
| --- | --- | --- |
| SLC22A1 | Metformin | Pharmacologic effect and PK |
| SLC22A2 | Metformin | Renal clearance | 
| SLC22A4 | Gabapentin | Renal clearance |
| CYP2B6 | Cylcophophamdie | Ovarian failure |
</script>
</section>
<section data-markdown>
<script type="text/template">
# Genes of Targets and Receptors
</script>
</section>
<section data-markdown>
<script type="text/template">
## Targets and receptors
| Gene | Drugs | Response affected|
| --- | --- | --- |
| ACE | ACEIs | Renoprotective effects, hypotension, cough |
| Thymidylate synthase | 5-FU | Colorectal cancer response |
| CCR5 | Antiretrovirals, interferon | Antiviral response |
| &Beta;<sub>2</sub> receptor (ADBR2) | &Beta;<sub>2</sub> blockers | Bronchodilation |
| &Beta;<sub>1</sub> receptor (ADBR1) | $beta;<sub>1</sub> blockers | BP, HR after &Beta;<sub>1</sub> blocker |
</script>
</section>
<section data-markdown>
<script type="text/template">
## Targets and receptors
| Gene | Drug | Response affected |
| --- | --- | --- |
| 5-lipoxygenase (ALOX5) | Leukotriene receptor antagonists | Asthma response |
| Dopamaine receptors (D<sub>2</sub>, D<sub>3</sub>, D<sub>4</sub>) | Haldoperidol, clozapine, thioridazine) | Antipsychotic response (D2, D3, D4), antipsychotic-induced tardive dyskinesia (D3), hyperprolactinemia in females (D2) |
| Serotonin receptor (5-HT<sub>2A</sub>) | Antipsychotics | Clozapine antipsychotic response , paroxetine antidepressant reponse |
</script>
</section>
<section data-markdown>
<script type="text/template">
## Glutatione-S-Transferase M1 (GSTM1)
A 50kb germline deletion which hass a frequency of 0.3-0.5% results in 50% reduction in glutathion conjugating
capacity in homozygous individuals compared to heterozygous individuals
</script>
</section>
<section data-markdown>
<script type="text/template">
## UGT1A1
The number of TA repeats in the promoter region of this gene influence quantitative expression levels in the liver.
TA repeats commonly exist as repeats of 6 or 7.
</script>
</section>
<section data-markdown>
<script type="text/template">
## CBS
A common 68bp indel has been associated with altered folate levels
</script>
</section>

        </div>
        <script src="../reveal/lib/js/head.min.js"></script>
        <script src="../reveal/js/reveal.js"></script>
        <script>
            // Full list of configuration options available at:
            // https://github.com/hakimel/reveal.js#configuration
            Reveal.initialize({
                controls: true,
                progress: true,
                history: true,
                center: true,
                transition: 'convex', // none/fade/slide/convex/concave/zoom
                // Optional reveal.js plugins
                dependencies: [
                    { src: '../reveal/lib/js/classList.js', condition: function() { return !document.body.classList; } },
                    { src: '../reveal/plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: '../reveal/plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
                    { src: '../reveal/plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
                    { src: '../reveal/plugin/zoom-js/zoom.js', async: true },
                    { src: '../reveal/plugin/notes/notes.js', async: true }
                ]
            });

        </script>
    </body>
</html>
